### DISCUSSION TITLE
Further development and maintence of the RTLSDR-Airband project

### DISCUSSION THREAD

**Discussion by charlie-foxtrot** [rtl-airband/RTLSDR-Airband#342] [General]
@szpajder has asked me to take over further development and maintence of the RTLSDR-Airband project, and as part of that has transferred the repo from https://github.com/szpajder/RTLSDR-Airband to https://github.com/charlie-foxtrot/RTLSDR-Airband

Hopefully this will be a seamless transition for everyone using the project as both the git and http URLs seem to auto-forward.

Thank you to @szpajder for sheepherding this project over the years and I hope to keep it alive and thriving for years to come!

**Comment by feelingwalnut** [comment]
thank you both for your work.

**Reply by LoganSound** [reply]
This is an incredible piece of software, thank you both for all the time it has taken to get it here.

**Comment by szpajder** [comment]
As many of you have probably noticed, there's been a little bit of a hiatus in the development of RTLSDR-Airband over the last year or so.  This is mostly due to the fact that my interests in radio hobby have shifted from voice channel monitoring to digital modes. In the recent years I've spun up a few projects related to this topic and I simply couldn't handle them all due to lack of time (I also don't have that much time for hobbies overall as I had before). So the choice was either to hand this project over to someone else or allow it to slowly rot and die. @charlie-foxtrot has made a few pretty significant contributions to the project over the years and is well acquainted with the codebase, so I think he will be a great maintainer. Kudos for taking the challenge!

I'll still be around and engage or maybe contribute something here, if time permits.

**Reply by om3wuc** [reply]
Hope this project will not die. Thanks @szpajder for your great work.

**Comment by itnavigate** [comment]
Thank you both!!
Just a suggestion - @charlie-foxtrot you may want to add notice of this change to the Readme.md.
I spent some time trying to get the the @szpajder repo before reading this conversation.🤦‍♂️The seamless redirects had me thinking something was broken at GitHub. 🤣

**Comment by barjac** [comment]
I have maintained rtlsdr-airband for Mageia for many years now and hope to continue.

Thanks for stepping in. :)

I was notified of the update but the tarball seems to not exist:


error: Could not download file https://github.com/charlie-foxtrot/RTLSDR-Airband/archive/v4.1.1/RTLSDR-Airband-4.1.1.tar.gz

EDIT - But now it does! Must have been a temporary glitch :)
